src/main/java/com/moneylendingapp/advice/ApiResponseEnvelope.java

```diff
@@ -0,0 +1,17 @@
+package com.moneylendingapp.advice;
+
+
+import lombok.Builder;
+import lombok.Data;
+
+import java.time.LocalDateTime;
+import java.util.List;
+
+@Data
+@Builder
+public class ApiResponseEnvelope {
+    private boolean successStatus;
+    private LocalDateTime timeStamp;
```

Comment:
timestamp is a mysql numenclature. maybe reqeuestDate or responseDate or something more useful and business